docker/musicbrainz-server-base/DBDefs.pm.m4

```diff
@@ -0,0 +1,128 @@
+include(`config.all.m4')dnl
+define(`_boolean_def', `subroutine'(``$1'', `ifelse'($1, ``1'', ``1'', ``0'')))dnl
+define(`_code_def', `subroutine'(``$1'', $1))dnl
+define(`_string_def', `subroutine'(``$1'', `quoted_string'(``$1'')))dnl
+define(`_wrap_def', `ifdef'(``$1'', ``$2''(```$1'''), ``dnl''))dnl
+define(`boolean_def', `_wrap_def'(``$1'', ``_boolean_def''))dnl
+define(`code_def', `_wrap_def'(``$1'', ``_code_def''))dnl
+define(`string_def', `_wrap_def'(``$1'', ``_string_def''))dnl
+define(`quoted_string', `changequote'(<VERY UNSAFE TEXT!>)`format'('%s', `patsubst'($1, ['\\], \\\&))`changequote')dnl
```

Comment:
Consider that a DBDefs configuration value from config.all.m4 might contain a `` ` `` or `'` which could be dangerous when substituted into the file...to work around that, we temporarily change the start quote to `<VERY UNSAFE TEXT!>`.